src/index.html

```diff
@@ -5,8 +5,294 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="nav">
+        <a class="logo" href="/">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <nav class="nav__box">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="is-active nav__item-link" href="#">apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__item-link" href="#">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__item-link" href="#">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__item-link" href="#"
+              data-qa="nav-hover">Laptops & Computers</a>
```

Comment:
You need to fix and check it everywhere
```suggestion
              <a
                class="nav__item-link"
                href="#"
                data-qa="nav-hover"
              >
                Laptops & Computers
              </a>
```